libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -17,12 +17,12 @@ export const addToReadingList = createAction(
   props<{ book: Book }>()
 );
 
-export const failedAddToReadingList = createAction(
+export const addToReadingListFailed = createAction(
   '[Reading List API] Failed add to list',
   props<{ book: Book }>()
 );
 
-export const confirmedAddToReadingList = createAction(
+export const addToReadingListConfirmed = createAction(
```

Comment:
If we're going for consistency, why stick with `Confirmed` here, over `Success` or `Completed`?

Reply:
renaming to `addToReadingListSucceeded`